developer/utils.py

```diff
@@ -0,0 +1,166 @@
+import os
```

Comment:
The pattern we have been adpoting so far is to not mix scripts and libraries. I personally find that quite confusing.

I suggest creating a subfolder under `developer` and putting an `__init__.py` in there just for clarity.